db/controllers/controller

```diff
@@ -50,9 +52,29 @@ exports.getAllArticles = (req, res, next) => {
 
 exports.getCommentsByArticleId = (req, res, next) => {
   const { article_id } = req.params;
-  fetchCommentsByArticleId(article_id)
-    .then((comments) => {
-      res.status(200).send(comments);
+  const idCheckQuery = checkIdExists(article_id, res, next);
+  const fetchCommentsQuery = fetchCommentsByArticleId(article_id, next);
+  Promise.all([idCheckQuery, fetchCommentsQuery])
+    .then((response) => {
+      if (response[0]) {
+        res.status(response[0].status).send(response[0].msg);
+      }
+      res.status(200).send(response[1]);
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.postCommentById = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+  return verifyDetails(article_id, username, res, next)
+    .then(() => {
+      return fetchPostedComment(article_id, username, body);
+    })
+    .then((comment) => {
+      res.status(201).send(`${username} commented ${comment}`);
```

Comment:
I can understand the want to send back the information in a user friendly fashion, but that is not the purpose of a server response. This data is intended for front end developers to use, and it's our job on backend to make it easy for them to work with. 
When sending a response, it should be sent as an object with an appropriate key name, for example:
`.send({ comment : comment})`